### ISSUE TITLE
Display of POI parking place

### ISSUE THREAD

**Issue by Berlin53** [osmandapp/OsmAnd#16413]
The new scheme for street parking was introduced in OpenStreetMap. Since then, when the POI overlay "parking place" is selected in OsmAnd, countless street-side parking bays are displayed as parking places. This makes the targeted search for central parking places difficult. Examples can be found in Berlin-Baumschulenweg.

The solution here would be not to display parking places with the key parking=street_side.

**Comment by Zirochkabila** [comment]
@Berlin53 The key parking=street_side is present in the amount of 293,546 - we cannot but display places with this tag.
What tag do you suggest as an alternative?

**Comment by Berlin53** [comment]
Hi Anatolii, I suggest not to show locations with this key (or also parking=lane, parking=on_kerb, parking=half_on_kerb, parking=shoulder) as parking place. Only parking places with the key amenity=parking without one of the additional keys parking=... are displayed, so that places described in the wiki Street parking do not appear as parking places.

 

Von: Anatolii ***@***.***> 
Gesendet: Freitag, 17. Februar 2023 17:12
An: osmandapp/OsmAnd ***@***.***>
Cc: Berlin53 ***@***.***>; Mention ***@***.***>
Betreff: Re: [osmandapp/OsmAnd] Display of POI parking place (Issue #16413)

 

@Berlin53 <https://github.com/Berlin53>  The key parking=street_side is present in the amount of 293,546 - we cannot but display places with this tag.
What tag do you suggest as an alternative?

**Comment by Zirochkabila** [comment]
@Berlin53 You're right, there are a lot of parking lots with the `parking=street_side` tag at https://www.openstreetmap.org/#map=19/52.46132/13.48201 
But, you can filter the type of parking you want to show on the map (shown in the screenshot) and the parking on the side of the road will not be displayed on the map

<table style="width:100%">
  <tr>
<th><img src="https://user-images.githubusercontent.com/104760013/221403564-a301f5e9-3f62-4d1b-9904-23bd425b357b.jpeg"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by Berlin53** [comment]
Hi Anatolii, thank you for your reply. Unfortunately, the filter does not help, because only very specific parking spaces are displayed here. A filter with parking spaces that are not displayed would be necessary. These are parking spaces with the keys parking=street_side, parking=lane, parking=on_kerb, parking=half_on_kerb, parking=shoulder. This filter should then also be able to be set up permanently.

 

Thanks

 

 

Von: Anatolii ***@***.***> 
Gesendet: Sonntag, 26. Februar 2023 10:51
An: osmandapp/OsmAnd ***@***.***>
Cc: Berlin53 ***@***.***>; Mention ***@***.***>
Betreff: Re: [osmandapp/OsmAnd] Display of POI parking place (Issue #16413)

 

@Berlin53 <https://github.com/Berlin53>  You're right, there are a lot of parking lots with the parking=street_side tag at https://www.openstreetmap.org/#map=19/52.46132/13.48201
But, you can filter the type of parking you want to show on the map (shown in the screenshot) and the parking on the side of the road will not be displayed on the map


 <https://user-images.githubusercontent.com/104760013/221403564-a301f5e9-3f62-4d1b-9904-23bd425b357b.jpeg>
